### DISCUSSION TITLE
Update yarn.lock

### DISCUSSION THREAD

**Discussion by Beraliv** [renovatebot/renovate#9539] [Request Help] [answered]
Hello! 👋

I'm interested if my `yarn.lock` can be updated

##  Currently what I see

PRs:
1. https://github.com/Beraliv/beraliv.dev/pull/71
2. https://github.com/Beraliv/beraliv.dev/pull/72

Only `package.json` is updated

## Repo

1. I use https://github.com/apps/renovate
2. I had only base config in https://github.com/Beraliv/beraliv.dev/blob/main/renovate.json 
3. I use yarn workspaces – https://github.com/Beraliv/beraliv.dev/blob/main/package.json#L7

## Next steps

I'm not sure what should I do to fix it:

1. I checked [lockFileMaintenance](https://docs.renovatebot.com/configuration-options/#lockfilemaintenance) and it's disabled by default (should I enable it or is it something different?)

2. I also found:

> WhiteSource Renovate will generate updated lock files such as package-lock.json or yarn.lock if you're already using them. It will also automatically resolve any conflicts after merges.

So should work for me I guess

3. Here https://github.com/renovatebot/renovate/pull/743 I see the support of yarn workspaces so should be fine

Please let me know where I'm wrong and what can I do to fix it?

**Comment by rarkins** [comment, marked as answer]
If you are using Yarn workspaces then why is there still a `yarn.lock` in the `packages/blog/` folder? I think this may be causing the problem, because if a `package.json` belongs to workspaces then it should share the root `yarn.lock`

**Reply by Beraliv** [reply]
I think I called `yarn` inside each folder that's why

Let me check if it helps

Thank you for the quick response!

**Reply by Beraliv** [reply]
Yeah, it worked! 🔥

Thank you again for the help!
